provisioning/miniprov/hare_mp/main.py

```diff
@@ -83,28 +83,48 @@ def _report_unsupported_features(features_unavailable):
         uf_db.store_unsupported_features('hare', features_unavailable))
 
 
-def logrotate():
+def get_server_type(url: str) -> str:
     try:
-        setup_info = get_data_from_provisioner_cli('get_setup_info')
-        if setup_info != 'unknown':
-            server_type = setup_info['server_type']
+        provider = ConfStoreProvider(url)
+        machine_id = provider.get_machine_id()
+        server_type = provider.get(
+            f'server_node>{machine_id}>type')
+
+        if server_type == 'VM':
+            return 'virtual'
+        else:
+            return 'physical'
+    except Exception as error:
+        logging.error('Cannot get server type (%s)', error)
+        return 'unknown'
+
+
+def logrotate(url: str):
+    try:
+        server_type = get_server_type(url)
+        logging.info('Server type (%s)', server_type)
+
+        if server_type != 'unknown':
             shutil.copyfile(
                 f'/opt/seagate/cortx/hare/conf/logrotate/{server_type}',
                 '/etc/logrotate.d/hare')
     except Exception as error:
         logging.error('Cannot configure logrotate for hare (%s)', error)
 
 
-def unsupported_feature():
+def unsupported_feature(url: str):
     try:
         features_unavailable = []
         path = '/opt/seagate/cortx/hare/conf/setup_info.json'
         with open(path) as hare_features_info:
             hare_unavailable_features = json.load(hare_features_info)
-            setup_info = get_data_from_provisioner_cli('get_setup_info')
```

Comment:
Sorry, I have basic questions.
My understanding is that Hare used to call a provisioner command "get_setup_info", which has now started failing, and it is suggested to Hare that now this info needs to be retrieved directly from ConfStore.
1. Why `get_setup_info` failed?
2. Isn't it expected that `get_setup_info` needs to be updated / corrected to fetch the data correctly? (as this command might be called from elsewhere as well)?

Reply:
@83bhp Any idea about the above comment from @suvratjoshi ?

Reply:
I think provisioner has deprecated get_setup_info and is no more supported, @83bhp, kindly confirm.

Reply:
We can keep this question open in public channels as discussed.
Approving this PR.